data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/CidrExpressionFunction.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import inet.ipaddr.IPAddress;
+import inet.ipaddr.IPAddressString;
+import org.opensearch.dataprepper.model.event.Event;
+
+import javax.inject.Named;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Named
+public class CidrExpressionFunction implements ExpressionFunction {
+
+    @Override
+    public String getFunctionName() {
+        return "isIPinCIDR";
```

Comment:
I think I preferred the last name and it is probably the most consistent. `isIpInCidr`.

Maybe there are other names we can use here as well?

* `isAddressInCidr`
* `cidrContains`

Reply:
Renamed to `cidrContains`